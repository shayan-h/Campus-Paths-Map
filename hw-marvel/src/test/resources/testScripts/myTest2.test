LoadGraph marvel marvel.csv

FindPath marvel IRON-MAN/TONY-STARK KANGAROO-II

FindPath marvel SILAS-GRANT ODIN-[ASGARDIAN]