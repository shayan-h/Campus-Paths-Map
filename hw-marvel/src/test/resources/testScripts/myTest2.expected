loaded graph marvel

path from IRON-MAN/TONY-STARK to KANGAROO-II:
IRON-MAN/TONY-STARK to ABSORBING-MAN/CARL-C via A-183
ABSORBING-MAN/CARL-C to KANGAROO-II via W2-164

path from SILAS-GRANT to ODIN-[ASGARDIAN]:
SILAS-GRANT to ODIN-[ASGARDIAN] via T-197